src/Apps/Collect/urlBuilder.ts

```diff
@@ -0,0 +1,64 @@
+import qs from "qs"
+import { State } from "./FilterState"
+
+export const buildUrlForCollectionApp = (state: State): string => {
+  const params = removeDefaultValues(state)
+  const queryString = qs.stringify(params)
+
+  return queryString
+    ? `${window.location.pathname}?${queryString}`
+    : window.location.pathname
+}
+
+export const buildUrlForCollectApp = (state: State): string => {
+  const fragment = buildCollectUrlFragmentFromState(state)
+
+  return fragment ? `/collect${fragment}` : "/collect"
```

Comment:
This is just a matter of opinion but I've found that through time code that doesn't return logic is a little easier to debug and maintain. Meaning:

```js
export const buildUrlForCollectApp = () => {
   const fragment = ... 
   const url = fragment ? `/collect${fragment}` : "/collect"
   return url
}
```
It also assigns contextual meaning to the return result that, when combined with the function name, makes things easier to parse for devs who wern't originally responsible for writing the code.

Reply:
No argument from me, I'll make this change.